src/properties/property-effects.html

```diff
@@ -1397,33 +1380,35 @@
        *   notification of change to a shared path).
        * @protected
        */
-      _setPathOrUnmanagedProperty(path, value, fromPath) {
+      _setPendingPropertyExternal(path, value, fromPath) {
```

Comment:
Add note about who calls this. This is the entry point where paths (slow path) can be switched on.